client/tests/webdriver/baseSpecs/searchFilters/taskFilter.spec.js

```diff
@@ -0,0 +1,30 @@
+import { expect } from "chai"
+import TaskFilter from "../../pages/searchFilters/taskFilter.page"
+
+describe("When using the task filter on the reports search", () => {
+  it("Should show all the tasks", async() => {
+    await TaskFilter.open()
+    await TaskFilter.openTaskFilter()
+
+    expect(await TaskFilter.getTaskCount()).to.be.equal(13)
```

Comment:
I think `expect(A).to.equal(B)` scans better than `expect(A).to.be.equal(B)`
[but not all existing assertions follow that pattern, alas]